src/usr/local/www/services_ntpd_acts.php

```diff
@@ -0,0 +1,323 @@
+<?php
+/*
+ * services_ntpd_acts.php
+ *
+ * part of pfSense (https://www.pfsense.org)
+ * Copyright (c) 2004-2016 Rubicon Communications, LLC (Netgate)
+ * Copyright (c) 2013 Dagorlad
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+##|+PRIV
+##|*IDENT=page-services-ntpd-acts
+##|*NAME=Services: NTP ACTS
+##|*DESCR=Allow access to the 'Services: NTP ACTS' page.
+##|*MATCH=services_ntpd_acts.php*
+##|-PRIV
+
+require_once("guiconfig.inc");
+
+if ($_POST) {
+	unset($input_errors);
+
```

Comment:
Input validation here is required, you will need to add code to verify that the input makes sense (proper type, within required ranges, items from a list exist in that list, etc)